### DISCUSSION TITLE
How to show have release notes show in PR for OSS repo?

### DISCUSSION THREAD

**Discussion by ahatzz11** [renovatebot/renovate#20438] [Request Help] [answered]
I am looking at having release notes from a public github repo for an OSS gradle plugin show up on the PRs to update said plugin. 

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

whitesource/renovate:3.0.0

### If you're self-hosting Renovate, select which platform you are using.

github.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

GitHub.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

We recently open sourced a gradle plugin of ours and published v1.1.0 to the gradle plugin portal. Renovate saw this publish and suggested an update to one of our internal repos, but no release notes were shown. 

Image of PR update with no release notes:
<img width="942" alt="image" src="https://user-images.githubusercontent.com/6256032/219106846-d6f5be51-966c-4dad-beb5-7e0d0a6925c2.png">

Public github repo with release notes for this 1.1.0 version: https://github.com/FigureTechnologies/gradle-semver-plugin/releases/tag/v1.1.0

Public gradle link: https://plugins.gradle.org/plugin/com.figure.gradle.semver-plugin

### What you tried so far.

- I tried to have a release on github that was an *exact* match to the package version, specifically trying `1.1.0` over `v1.1.0`, but this didn't work.
- I validated that our vcs metadata in our artifact is correct with `vcsUrl`: https://github.com/FigureTechnologies/gradle-semver-plugin/blob/main/build-logic/build-conventions/src/main/kotlin/local.figure.publishing.gradle.kts#L20

---

We own the source repo and the publishing of this plugin, so I'm happy to try whatever to get this working!

**Comment by rarkins** [comment, marked as answer]
I think this is the same topic as #19514

**Reply by ahatzz11** [reply]
Yes this is the same issue, thanks for the link and quick response! Definitely sad that gradle plugin portal strips some of that data from pom.

Is adding support for some of the gradle metadata something renovate would be open to since the plugin portal seems to kill off metadata?

**Reply by rarkins** [reply]
The community can contribute them here: https://github.com/renovatebot/renovate/blob/main/lib/modules/datasource/metadata-manual.ts

**Reply by ahatzz11** [reply]
@rarkins awesome thanks! I made a contribution, I'd appreciate a review on it since I'm not 100% sure how it should be done for gradle https://github.com/renovatebot/renovate/pull/20441
